pkgs/development/tools/rust/cargo-call-stack/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-call-stack";
+  version = "0.1.6";
+
+  src = fetchFromGitHub {
+    owner = "japaric";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "00ka0yx3114wmpngs44fv2j3r6jxh1v1w9kzzmbklynpi3wmb65b";
+  };
+
+  cargoPatches = [ ./add-Cargo-lock.patch ];
```

Comment:
Could we use fetchCrate to avoid this?

Reply:
there is also a new release that has a Cargo.lock